snap/hooks/install

```diff
@@ -5,9 +5,11 @@
 cp "${SNAP}"/config/etcd.conf.yml "${SNAP_DATA}"/etcd.conf.yml
 chown -R 584788:584788 "${SNAP_DATA}"/*
 
-# create the data directory
+# create the data directory only if not exists
 DATA="${SNAP_COMMON}"/var/lib/etcd
-mkdir -p "${DATA}"
+if [ ! -d "${DATA}" ]; then
+  mkdir -p "${DATA}"
+fi
```

Comment:
hmm, that was already the case before due to the `-p` flag on `mkdir`. It would have not effectively be re-created if it existed. Making this check redundant